website/content/docs/templates/hcl_templates/contextual-variables.mdx

```diff
@@ -145,6 +147,22 @@ source "amazon-ebs" "cannonical-ubuntu-server" {
 }
 ```
 
+# HCP Packer Version Fingerprint
+
+If your build is pushing metadata to the HCP Packer registry, this variable is
+set to the value of the Version Fingerprint associated with this run.
+
+```hcl
+source "amazon-ebs" "cannonical-ubuntu-server" {
+  ami_name         = "packer-example"
+  // ...
+  run_volume_tags = {
+    hcp_version_fingerprint = packer.versionFingerprint
```

Comment:
I wonder, does that work? I'm not certain the fingerprint will be instanciated by the time we evaluate this.

Reply:
To be clear if it doesn't we should fix it later, but I'm cautious giving this example if it doesn't work